### PR TITLE
[CI:DOCS] RELEASE_PROCESS.md: cosmetic fix

### PR DESCRIPTION
**Pull request by lsm5**
Follow up to pr#13416

Insert line break to get rid of the horizontal scroll bar.

Resolves: https://github.com/containers/podman/pull/13416#discussion_r818991807

Signed-off-by: Lokesh Mandvekar <lsm5@fedoraproject.org>

@rhatdan @ashley-cui @mheon @cevich PTAL

### CODE REVIEW

File: RELEASE_PROCESS.md [hunk 1]
```diff
@@ -241,7 +241,8 @@ spelled with complete minutiae.
 
       ```shell
       $ git checkout vX.Y.Z
-      $ make podman-remote-release-darwin_amd64.zip podman-remote-release-darwin_arm64.zip podman-remote-release-windows_amd64.zip podman.msi podman-remote-static
+      $ make podman-remote-release-darwin_amd64.zip podman-remote-release-darwin_arm64.zip \
```
Comment (ashley-cui) [inline RELEASE_PROCESS.md:244]: Might as well
```suggestion
      $ make podman-remote-release-darwin_amd64.zip \
                   podman-remote-release-darwin_arm64.zip \
                   podman-remote-release-windows_amd64.zip \
                   podman.msi \
                   podman-remote-static
```

Comment (rhatdan) [reply RELEASE_PROCESS.md:244]: I like that better.

**Comment by TomSweeneyRedHat** [comment]
LGTM once, @ashley-cui 's nit is picked.

File: RELEASE_PROCESS.md (hunk 1 above)
Comment (cevich) [reply RELEASE_PROCESS.md:244]: +1

Comment (lsm5) [reply RELEASE_PROCESS.md]: @ashley-cui done, credited you.

**Comment by lsm5** [comment]
PTAL.

**Comment by ashley-cui** [comment]
LGTM!

**Pull request merged**